jupyter_elfinder/elfinder.py

```diff
@@ -1439,30 +1439,31 @@ def __put(self) -> None:
 
     def __archive(self) -> None:
         """Compress files/directories to archive."""
-        if (
-            not self._options["archivers"]["create"]
-            or API_TYPE not in self._request
-            or API_TARGET not in self._request
-            or API_TARGETS not in self._request
-        ):
+        # TODO: We don't support "name" field yet.
+        # "name" is a parameter according to api 2.1.
```

Comment:
Are the docs correct here?

https://github.com/Studio-42/elFinder/wiki/Client-Server-API-2.1#archive